### ISSUE TITLE
Unable to get the output in json for cluster list

### ISSUE THREAD

**Issue by vamsigutta** [IBM-Cloud/ibm-cloud-cli-sdk#359]
Hello Team,

The issues is reproducing to me in the version **2.14.0** . The issue is the that the list of the cluster is not reported in json inspite of the presence of the clusters. Please find the screenshot below for reference
<img width="905" alt="Screenshot 2023-02-02 at 2 41 43 PM" src="https://user-images.githubusercontent.com/19907690/216285088-bcb366d2-c258-4256-bb8b-031d64878a1b.png">

**Comment by steveclay** [comment]
@vamsigutta is this the command you are attempting that is not producing json for you? `ibmcloud ks cluster ls --output json`

**Comment by vamsigutta** [comment]
yes @steveclay it returns an empty array` []%` as a result rather than the list of clusters. While without json flag I can visualize the existing clusters
